app/src/main/java/co/nimblehq/compose/crypto/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,65 @@
+package co.nimblehq.compose.crypto.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Extend final [Colors] class to provide more custom app colors.
+ */
+data class AppColors(
+    val themeColors: Colors,
+
+    val coinChartLineDrawer: Color = CaribbeanGreen,
+    val coinChartLineShaderGradient: List<Color> = listOf(CaribbeanGreenAlpha30, Transparent),
+    val coinChartTimeIntervalBackground: Color = Transparent,
+    val coinChartTimeIntervalBackgroundSelected: Color = CaribbeanGreen,
+    val coinChartTimeIntervalText: Color = OsloGray,
+    val coinChartTimeIntervalTextSelected: Color = White,
+    val coinInfoAppBarIconTint: Color,
+    val coinInfoSellBuyBackground: Color,
+    val coinInfoSellBuyText: Color = White,
+    val coinItemBackground: Color,
+    val coinNameText: Color,
+
+    val buttonBlueBackground: Color = TiffanyBlue,
+    val buttonRedBackground: Color = FireOpal,
+
+    val portfolioCardBackgroundGradient: List<Color> = listOf(MetallicSeaweed, TiffanyBlue),
+    val portfolioCardShadow: Color = TiffanyBlue,
+    val priceChangeButtonBackground: Color = Water,
+    val priceChangeButtonBackgroundInDetail: Color = WhiteIce,
+    val priceTextNegative: Color = FireOpal,
+    val priceTextPositive: Color = GuppieGreen,
+    val pullRefreshBackground: Color,
+    val pullRefreshContent: Color = CaribbeanGreen,
+
+    val text: Color,
+    val textSectionLink: Color = TiffanyBlue,
+)
+
+internal val LightColorPalette = AppColors(
+    themeColors = lightColors(
+        surface = Guyabano,
+    ),
+    coinInfoAppBarIconTint = Quartz,
+    coinInfoSellBuyBackground = White,
+    coinItemBackground = White,
+    coinNameText = SonicSilver,
+    pullRefreshBackground = White,
+    text = DarkJungleGreen
```

Comment:
@luongvo If we follow MaterialTheme, do you think we should follow the naming convention for the app global scope? `primary, secondary, accent` 🤔

Reply:
@hoangnguyen92dn As I explained in PR insight, `AppColors` includes `themeColors` which references to MaterialTheme colors. In case we need to do any theme color updates, we should apply them in this block.

```
    themeColors = darkColors(
        surface = DarkJungleGreen
    ),
```

<img width="787" alt="image" src="https://user-images.githubusercontent.com/16315358/202072531-7bba71e7-2194-4deb-a962-b3a2276765a3.png">